.gitignore

```diff
@@ -1 +1,19 @@
-.idea/
\ No newline at end of file
+./.idea
```

Comment:
I'm not sure if this is intentional, but in my case .idea is placed in the same dir as .git
please add .idea